app/src/main/AndroidManifest.xml

```diff
@@ -131,108 +124,80 @@
-            android:configChanges="orientation|keyboardHidden|screenSize">
-        </activity>
-
+            android:configChanges="orientation|keyboardHidden|screenSize"
+            android:label="Summary"></activity>
         <activity
             android:name=".ui.OrderConfirmActivity"
-            android:label="Shipping Address"
-            android:configChanges="orientation|keyboardHidden|screenSize">
-        </activity>
-
+            android:configChanges="orientation|keyboardHidden|screenSize"
+            android:label="Shipping Address"></activity>
         <activity
             android:name=".ui.paymentGateway.PaymentModeActivity"
-            android:label="Select Payment Mode"
-            android:configChanges="orientation|keyboard|screenSize">
-        </activity>
-
+            android:configChanges="orientation|keyboard|screenSize"
+            android:label="Select Payment Mode"></activity>
         <activity
             android:name=".ui.paymentGateway.CreditDebitCardActivity"
-            android:label="Enter Details"
-            android:configChanges="orientation|keyboard|screenSize">
-        </activity>
-
+            android:configChanges="orientation|keyboard|screenSize"
+            android:label="Enter Details"></activity>
         <activity
             android:name=".ui.paymentGateway.NetBankingActivity"
-            android:label="Select Bank"
-            android:configChanges="orientation|keyboard|screenSize">
-        </activity>
-
-        <activity android:name=".ui.paymentGateway.PaymentsActivity"
             android:configChanges="orientation|keyboard|screenSize"
-            android:windowSoftInputMode="stateAlwaysHidden">
-        </activity>
-
+            android:label="Select Bank"></activity>
+        <activity
+            android:name=".ui.paymentGateway.PaymentsActivity"
+            android:configChanges="orientation|keyboard|screenSize"
+            android:windowSoftInputMode="stateAlwaysHidden"></activity>
         <activity
             android:name=".ui.ProfileDetailsActivity"
             android:configChanges="orientation|keyboard|screenSize"
-            android:label="Profile Details">
-        </activity>
-
+            android:label="Profile Details"></activity>
         <activity
             android:name=".ui.ProfilePhotoActivity"
             android:configChanges="orientation|keyboard|screenSize"
-            android:label="Profile Photo">
-        </activity>
-
+            android:label="Profile Photo"></activity>
         <activity
             android:name=".ui.CropImageActivity"
             android:configChanges="orientation|keyboard|screenSize"
-            android:label="Crop Image">
-        </activity>
+            android:label="Crop Image"></activity>
+        <activity
+            android:name=".authenticator.ResetPasswordVerificationActivity"
+            android:label="@string/title_activity_reset_password_verification"></activity>
```

Comment:
All you need to do was to add only these 3 lines. And you went ahead and made a complete code cleanup of removing and adding spaces, empty lines etc. Please don't do like this hereafter.